packages/discovery-provider/src/queries/get_track_signature.py

```diff
@@ -157,7 +157,19 @@ def get_track_download_signature(args: GetTrackDownloadSignature):
         authed_user_wallet = authed_user["user_wallet"].lower()
         if signature_user_wallet != authed_user_wallet:
             return None
-        return {"signature": signature_obj, "cid": cid, "filename": filename}
+        # todo: use commented line below once logic is added to pass in correct signature
```

Comment:
related [linear card](https://linear.app/audius/issue/PAY-2365/allow-losslessoriginal-download-for-nft-gated-tracks). i'll take care of it next week